tools/wpt/run.py

```diff
@@ -287,6 +287,9 @@ def setup_kwargs(self, kwargs):
         # Shorten delay for Reporting <https://w3c.github.io/reporting/>.
         kwargs["binary_args"].append("--short-reporting-delay")
 
+        # Point all .test domains to localhost for Chrome
+        kwargs["binary_args"] += ["--host-resolver-rules=MAP nonexistent.*.test ~NOTFOUND, MAP *.test 127.0.0.1"]
```

Comment:
I think this should go in `tools/wptrunner/wptrunner/browsers/chrome.py`

Reply:
Do you mean this one specifically, or all of the binary_args appended here?
Seems `chrome.py` doesn't do anything with binary_args right now, all of it gets updated here.

Could you point to an example?

Reply:
Apologies for using the gecko sync, but the search tools are better there ;) 

https://searchfox.org/mozilla-central/source/testing/web-platform/tests/tools/wptrunner/wptrunner/browsers/chrome.py#59-67

is where the binary kwargs are actually passed down to ChromeDriver. For `--headless` we add an argument there. In this case I'd expect that we just set the args unconditionally i.e. replace the above code with something like:

```python
    chrome_options = capabilities["goog:chromeOptions"]

    chrome_options["args"] = ["--host-resolver-rules=MAP nonexistent.*.test ~NOTFOUND, MAP *.test 127.0.0.1"]
    if kwargs["binary_args"] is not None:
        chrome_options["args"].extend(kwargs["binary_args"])
    if kwargs["headless"] and "--headless" not in kwargs["binary_args"]:
        chrome_options["args"].append("--headless")

     if kwargs["binary"] is not None:
        chrome_options["binary"] = kwargs["binary"]
 ```

Reply:
Thanks for the pointer, done.

I also moved the handful of other flags that are unconditionally set so they're all in the same place.